plugin/callgraph/signatures.go

```diff
@@ -0,0 +1,145 @@
+package callgraph
+
+import (
+	_ "embed"
+	"strings"
+
+	"github.com/safedep/code/core"
+	"github.com/safedep/dry/ds/trie"
+	"github.com/safedep/dry/log"
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed signatures.yaml
+var signatureYAML []byte
```

Comment:
We need to decouple the signatures. Once the signatures specs are defined, the matcher should take a list of signatures to match